### PR TITLE
YBRO Argothian Uprooting & Foundry Groundbreaker

### PR DESCRIPTION
**Pull request by Simisays**
Should be good to merge

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/argothian_uprooting.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Argothian Uprooting
+ManaCost:X G
+Types:Sorcery
+A:SP$ PutCounter | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Land.YouCtrl | TgtPrompt$ Select up to X lands you control | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBAnimate | SpellDescription$ Put two +1/+1 counters on each of up to X target lands you control. They each become 0/0 Elemental creatures with reach, haste, and “When this creature leaves the battlefield, conjure a card named Forest onto the battlefield tapped.” They're still lands.
+SVar:DBAnimate:DB$ Animate | Defined$ ParentTarget | Power$ 0 | Toughness$ 0 | Types$ Creature,Elemental | Keywords$ Haste & Reach | Duration$ Permanent | Triggers$ DiesTrig
+SVar:DiesTrig:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigConjure | TriggerDescription$ When this creature leaves the battlefield, conjure a card named Forest onto the battlefield tapped
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/argothian_uprooting.txt:6]: Wrong Destination

File: forge-game/src/main/java/forge/game/ability/effects/MakeCardEffect.java [hunk 2]
```diff
@@ -117,6 +117,9 @@ public void resolve(SpellAbility sa) {
                 if (sa.hasParam("FaceDown")) {
                     made.turnFaceDown(true);
                 }
+                if (sa.hasParam("Tapped")) {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/MakeCardEffect.java:120]: Should be above moveTo

Comment (Simisays) [reply forge-game/src/main/java/forge/game/ability/effects/MakeCardEffect.java]: Thanks you

**Review by tool4ever — APPROVED** [review]

File: forge-game/src/main/java/forge/game/ability/effects/MakeCardEffect.java [hunk 3]
```diff
@@ -95,6 +95,9 @@ public void resolve(SpellAbility sa) {
                         game.getAction().moveTo(ZoneType.None, card, sa, moveParams);
                         cards.add(card);
                         toMake--;
+	                        if (sa.hasParam("Tapped")) {
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/ability/effects/MakeCardEffect.java:98]: the code formatting for this is a bit off, @tool4ever or another might fix that later with another MR

**Pull request merged**